### ISSUE TITLE
Add template.html format to sub pages folder

### ISSUE THREAD

**Issue by Shannspan** [codelikeagirlau/accessibility-resources#48]
Subpages folder:

-  Contains accessibility-courses.html and web-developer-pledge.html
-  accessibility-courses.html doesn't have to header and nav from the html template
-  One PR attempted but the changes made are not correct (notes made on the PR - please reject)
-  Will attempt correct changes as a new PR
-  Not sure why but changes from the template are not sitting correctly (though the nav now appears)
-  Will investigate this further so that nav is a sidebar as per the template and submit further PR's when complete

**Comment by Shannspan** [comment]
This has now been superceded by issue #57
